wp/plugin/wp.ml

```diff
@@ -18,149 +18,158 @@ module Typ = Extension.Type
 
 let name = "wp"
 
-let doc = "WP is a comparative analysis tool. It can compare two binaries \
-           and check that they behave in similar ways. It can also be used \
-           to check a single binary for certain behaviors."
+let doc = {|WP is a comparative analysis tool. It can compare two binaries
+            and check that they behave in similar ways. It can also be used
+            to check a single binary for certain behaviors.|}
 
 (* Mandatory arguments. *)
 
 let files = Cmd.arguments Typ.file
-    ~doc:"Path(s) to one or two binaries to analyze. If two binaries are \
-          specified, WP will run a comparative analysis."
+    ~doc:{|Path(s) to one or two binaries to analyze. If two binaries are
+           specified, WP will run a comparative analysis. If 0 or more than 2
+           files are give, WP will fail.|}
```

Comment:
This might be my personal preference, but the phrase "WP will fail" is ambiguous, and as a user, I think, "what does that mean? Will it fail silently? Will it throw an error? Will it give me false positives?" 

If it throws an error, I think it would be better to say "WP will throw an error".

Reply:
WP will print out an error message and then exit. I can replace this with "If 0 or more than 2 files are given, WP will not run an analysis and will exit."

Reply:
I think "WP will exit with an error message" is just fine.

Reply:
Always coming up with the nice and concise documentation. :)